Ginger/Ginger/SolutionWindows/TreeViewItems/BusinessFlowTreeItem.cs

```diff
@@ -103,7 +103,7 @@ void ITreeViewItem.SetTools(ITreeView TV)
                     TreeViewUtils.AddMenuItem(mContextMenu, "Automate", Automate, null, eImageType.Automate);
                 }
                 
-                AddItemNodeBasicManipulationsOptions(mContextMenu);
+                AddItemNodeBasicManipulationsOptions(mContextMenu, allowActionConversion: true);
```

Comment:
i asked on previous review to remove convertsion from the TreeViewItemGenericBase because it belong only to BusinessFlow tree items and not to all

Reply:
i kept it since it was required to handle in 2 classes (BusinessFlowTreeItem, BusinessFlowsFolderTreeItem), so added in "TreeViewItemGenericBase" also checked it was not displayed in other "TreeView". Now changed it and added in Business tree item classes.